packages/commonwealth/server/middleware/lookupAddressIsOwnedByUser.ts

```diff
@@ -71,12 +71,21 @@ const lookupAddressIsOwnedByUser = async (
     return [null, 'Must provide an address'];
   }
 
+  const query: {
+    chain: string;
+    address: string;
+    user_id?: number;
+  } = {
+    chain: req.body.author_chain,
+    address: req.body.address,
+  };
+
+  if (!req.user.isAdmin) {
```

Comment:
bit idiosyncratic because of how validateAuthor works; basically, if the user who made the request is a god mode admin, they won't be included in the query for the resource because they do NOT need to be the author of the comment (or thread, or wtv) in order to take an action on it. This line just adds them to the query to check for authorship if they aren't a god mode admin.